core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -476,7 +476,7 @@ private void updateMetrics(boolean success, Instant startTime,
    * be synced as well.
    *
    * @param path The path to sync
-   * @return true if this path was synced
```

Comment:
The comment is badly written, it should be return true if the path was synced as the result of this call.  the original implementation is correct.

Reply:
Do you mean #15265 is not a bug?